services/spar/schema/src/V7.hs

```diff
@@ -0,0 +1,18 @@
+module V7 (migration) where
+
+import Imports
+import Cassandra.Schema
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 7 "Store default SSO code" $ do
+
+    -- partition_key_always_default should always be "default".
+    -- It exists so the row is always at a known partition.
+    void $ schema' [r|
+        CREATE TABLE if not exists default_idp
+            ( partition_key_always_default text
```

Comment:
not sure about the name.  this restricts us to never give it another value in the future, without adding any strong guarantees (it's just a name, same weight as a comment).

Reply:
We can make this table a more general "settings" key-value store, but the values can only be UUIDs now. I think I'd rather make a separate table for different things.

And this is slightly better than just a comment, since everyone writing a query has to type it out. If you want you could say it's an enforced per-usage-site comment. ;)

Reply:
good points.